website/content/docs/api-gateway/tech-specs.mdx

```diff
@@ -18,9 +18,10 @@ Verify that your environment meets the following requirements prior to using Con
 Your datacenter must meet the following requirements prior to configuring the Consul API Gateway:
 
 - Kubernetes 1.21+
+  - Kubernetes 1.24 is not supported at this time.
 - `kubectl` 1.21+
-- Consul 1.12.0+
-- HashiCorp Consul Helm chart 0.43.0+
+- Consul 1.11.2+
```

Comment:
@im2nguyen is this the correct version? Is this not for Consul 1.12?

Reply:
this is the correct version. this is taken from teh changes suggested by @Jeff-Apple 

https://github.com/hashicorp/consul/pull/13427/files